src/App.js

```diff
@@ -1,10 +1,27 @@
 import React from "react";
+import MainLayout from "./components/layout/MainLayout/MainLayout";
+import Navigation from "./components/layout/Navigation/Navigation";
+import Header from "./components/layout/Header/Header";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 
 const App = () => {
   return (
     <>
-      <h1>Welcome to Coders Camp 2021</h1>
-      <h2>SPA Project</h2>
+      <BrowserRouter>
+        <Navigation />
+        <MainLayout>
+          <Routes>
```

Comment:
add separatate folder under `/src` for routes (`/src/routes`) and create routes agregator there